core/domain/feedback_jobs_one_off.py

```diff
@@ -100,3 +103,64 @@ def map(item):
     @staticmethod
     def reduce(key, value):
         pass
+
+
+class SuggestionMigrationOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for converting all suggestions from the old model to the
+    new model.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [feedback_models.SuggestionModel]
+
+    @staticmethod
+    def map(suggestion):
+        new_suggestion_id = (
+            suggestion_models.TARGET_TYPE_EXPLORATION + '.' + suggestion.id)
+        thread = feedback_models.FeedbackThreadModel.get_by_id(suggestion.id)
+        if thread.status == feedback_models.STATUS_CHOICES_OPEN:
+            status = suggestion_models.STATUS_RECEIVED
+        elif thread.status == feedback_models.STATUS_CHOICES_FIXED:
+            status = suggestion_models.STATUS_ACCEPTED
+        elif thread.status == feedback_models.STATUS_CHOICES_IGNORED:
+            status = suggestion_models.STATUS_REJECTED
+
+        score_category = (
+            suggestion_models.SCORE_TYPE_CONTENT +
+            suggestion_models.SCORE_CATEGORY_DELIMITER +
+            exp_services.get_exploration_by_id(
+                suggestion.exploration_id).category)
+
+        # The change_cmd has an old value field which will be set at the time of
+        # accepting the suggestion. For already accepted suggestions of the old
+        # system, this value cannot be determined now. As the suggestions will
```

Comment:
Perhaps, it is much more clear if we say that "We do not have any suggestions for translations, hence the migration job will not be dealing with such suggestions. So while migrating, we can set the translations field as {} without losing any data"

Reply:
Done Thanks!